contracts/deploy/010_upgrade_single_asset_staking.js

```diff
@@ -0,0 +1,75 @@
+//
+// Script to upgrade the Single Asset Staking contract.
+//
+const {
+  isMainnet,
+  isFork,
+  isMainnetOrRinkebyOrFork,
+} = require("../test/helpers.js");
+const {
+  log,
+  deployWithConfirmation,
+  executeProposal,
+} = require("../utils/deploy");
+const { proposeArgs } = require("../utils/governor");
+const { getTxOpts } = require("../utils/tx");
+
+const deployName = "007_upgrade_single_asset_staking";
+
+const upgradeSingleAssetStaking = async ({ getNamedAccounts }) => {
+  console.log(`Running ${deployName} deployment...`);
+
+  const { governorAddr } = await getNamedAccounts();
+  const sGovernor = ethers.provider.getSigner(governorAddr);
+
+  //
+  // Deploy the new implementation.
+  //
+  const dSingleAssetStaking = await deployWithConfirmation(
+    "SingleAssetStaking"
+  );
+
+  //
+  // Upgrade.
+  //
+  const cOGNStakingProxy = await ethers.getContract("OGNStakingProxy");
+  const propDescription = "OGNStaking upgrade";
+  const propArgs = await proposeArgs([
+    {
+      contract: cOGNStakingProxy,
+      signature: "upgradeTo(address)",
+      args: [dSingleAssetStaking.address],
+    },
+  ]);
+
+  if (isMainnet) {
+    // On Mainnet upgrade has to be handled manually via a multi-sig tx.
+    log(
```

Comment:
I recently changed our pattern here so that we don't need to run this propose step separately.
Do you mind calling `await sendProposal(propArgs, propDescription);` here?
See <https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/deploy/009_ousd_fix.js#L71>

Reply:
Have done this change